std/algorithm/sorting.d

```diff
@@ -2889,18 +2889,46 @@ void partialSort(alias less = "a < b", SwapStrategy ss = SwapStrategy.unstable,
     Range)(Range r, size_t n)
     if (isRandomAccessRange!(Range) && hasLength!(Range) && hasSlicing!(Range))
 {
-    topN!(less, ss)(r, n);
-    sort!(less, ss)(r[0 .. n]);
+    partialSort!(less, ss)(r[0 .. n], r[n .. $]);
 }
 
 ///
-@safe unittest
+unittest
 {
     int[] a = [ 9, 8, 7, 6, 5, 4, 3, 2, 1, 0 ];
     partialSort(a, 5);
     assert(a[0 .. 5] == [ 0, 1, 2, 3, 4 ]);
 }
 
+/**
+Stores the smallest elements of the two ranges in the left-hand range in sorted order.
```

Comment:
This description isn't very descriptive compared to the original description of `partialSort` :(
I know it's copy-pasted from `topN` ...
I would prefer if it's `ditto`ed with the previous documentation, s.t. the user also gets the information about expected run time etc.